### PR TITLE
LIT2382 6396

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
An update for the Miracles of Jesus and the Introduction.

### CODE REVIEW

File: 2001-3000/LIT2382Taamme.xml [hunk 1]
```diff
@@ -115,7 +115,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
          <div type="edition">
             <div type="textpart" xml:id="Introduction" corresp="LIT6396MJIntroduction">
-               <label>Introduction, Grébaut 1917 No 1</label>
+               <label>Introduction, Grébaut 1917 No. 1 (1-6)</label>
```
Comment (thea-m) [inline 2001-3000/LIT2382Taamme.xml:118]: Could you mark up the reference, here and later?

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by DenisNosnitsin1970** [comment]
I will add number of pages ot miracles 31-40 as soon as I reach the publication.

**Comment by thea-m** [comment]
@DenisNosnitsin1970 just so that we don't forget it. If you have the opportunity, could you add the links?

**Comment by DenisNosnitsin1970** [comment]
Yes.

**Comment by DenisNosnitsin1970** [comment]
Sorry, at this point I have to ask, which links are meant?

**Comment by thea-m** [comment]
I'm sorry, I meant the page numbers of the miracles 31-40. No idea what made me write links.

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
the pages are added but there is something wrong with this branch, I will remove 6396 from here and commit it separately

**Comment by thea-m** [comment]
I still don't see the page numbers for miracles 31-40

**Comment by DenisNosnitsin1970** [comment]
I think they are not published, if I am not mistaken.

**Comment by eu-genia** [comment]
Yes I have merged all

**Comment by eu-genia** [comment]
You mean page numbers? sorry I thought they were, if there is more to edit please create a new branch/PR

**Comment by eu-genia** [comment]
The current version has numbers up to number 30

**Comment by DenisNosnitsin1970** [comment]
I added for those which are edited. For the rest only numbers from the article.

**Pull request merged**